next-tavla/app/(admin)/components/CreateBoard/index.tsx

```diff
@@ -83,13 +83,23 @@ function CreateBoard() {
                             const organization = data.get(
                                 'organization',
                             ) as TOrganizationID
-
+                            const privateBoardCheckbox =
+                                data.get('check-privateBoard')
```

Comment:
dette navnet er litt for langt og knotete, kan hete "private" for eksempel

Reply:
får dette når jeg bytter til private `'private' is a reserved word in strict mode. Modules are automatically in strict mode.ts(1214)`